### PR TITLE
Content updates for auto-generated emails

### LINKED ISSUE #1286: Content Review: Auto-generated emails
### Issue description

Review the emails identified within the "Auto-emails" section of [this spreadsheet](https://docs.google.com/spreadsheets/d/18wwmEioSr6BU9Y6G6ihqzS7P1OaE65Ifoipg9-Gxcjo/edit#gid=710884559).

While reviewing those emails, also review the content that appears on any linked pages that lead to beta.get.gov. For example, some of the emails link to "help" pages.

Use the [Content Review Checklist](https://docs.google.com/document/d/13JrJZOAQCkt-G5zKiIzBoNg_4ZJm_-FxGH2aN2fS86s/edit#heading=h.41lgwrhbhke5) as a guide for doing the review.

### Acceptance criteria

- [ ] All auto-generated emails have been reviewed and updates have been made in GitHub.
- [ ] All linked pages that lead to beta.get.gov have also been reviewed and updates have been made in GitHub
- [ ] The status for each reviewed page is updated in [this spreadsheet](https://docs.google.com/spreadsheets/d/18wwmEioSr6BU9Y6G6ihqzS7P1OaE65Ifoipg9-Gxcjo/edit#gid=1246252690) to indicate the outcome of the review.
- [ ] Any dev tickets identified during the review have been created.

### Additional context

Note: Determine where all "Contact us" links should point and ensure they are consistent across emails. 

### Links to other issues

Blocked by #1248

### PR DESCRIPTION
**Pull request by Katherine-Osos**
## Ticket

Resolves #1286 

## Changes

Content updated to match changes made in the corresponding Google doc for each auto-generated email: 

- [Domain invitation email](https://docs.google.com/document/d/1HnMdniAB9INphrLwKjiACyNLrenyXy0QFf0-yz4EStI/edit) (body text only)
- [Submission email](https://docs.google.com/document/d/1edfim3cHbyBeYmlbpIjPXB2uAL8qlyC6M05AxiRRkwM/edit) (body text and subject line)
- [Withdrawn request](https://docs.google.com/document/d/1mMdBHOSrqWVsOlBuFd3BItdUnbGbsllJAxkDOujaSl4/edit) (body text and subject line)
- [Approved email](https://docs.google.com/document/d/1fIl752pZrDwm4SxF1IXKByYyXPmt97YiMJx0a3sQt3g/edit) (body text and subject line)
- [Rejected email](https://docs.google.com/document/d/1zohOnNTMs6kw_c06NCYx11T9wgekjrt8apv3_SbMlGM/edit) (body text and subject line)

You can review by triggering the emails thru my sandbox [ko](https://getgov-ko.app.cloud.gov/). Or I can trigger the emails for you - no problem.

### CODE REVIEW

File: src/registrar/templates/emails/domain_invitation.txt [hunk 1]
```diff
@@ -3,30 +3,36 @@ Hi.
...
 {{ requester_email }} has added you as a manager on {{ domain.name }}.
 
+You can manage this domain on the .gov registrar. <https://manage.get.gov>
+
+
 YOU NEED A LOGIN.GOV ACCOUNT
 You’ll need a Login.gov account to manage your .gov domain. Login.gov provides
 a simple and secure process for signing into many government services with one
-account. If you don’t already have one, follow these steps to create your
+account.
```
Comment (Katherine-Osos) [inline src/registrar/templates/emails/domain_invitation.txt:12]:
```suggestion
account
```

File: src/registrar/templates/emails/domain_invitation.txt [hunk 2]
```diff
@@ -3,30 +3,36 @@ Hi.
 
 {{ requester_email }} has added you as a manager on {{ domain.name }}.
 
+You can manage this domain on the .gov registrar. <https://manage.get.gov>
```
Comment (Katherine-Osos) [inline src/registrar/templates/emails/domain_invitation.txt:6]:
```suggestion
You can manage this domain on the .gov registrar <https://manage.get.gov>
```

File: src/registrar/templates/emails/status_change_approved.txt [hunk 3]
```diff
@@ -5,36 +5,37 @@ Congratulations! Your .gov domain request has been approved.
...
 DOMAIN REQUESTED: {{ application.requested_domain.name }}
 REQUEST RECEIVED ON: {{ application.submission_date|date }}
-REQUEST #: {{ application.id }}
-STATUS: In review
+STATUS: Approved
 
-Now that your .gov domain has been approved, there are a few more things to do before your domain can be used. 
+You can manage your approved domain on the .gov registrar. <https://manage.get.gov>
 
+----------------------------------------------------------------
```
Comment (Katherine-Osos) [inline src/registrar/templates/emails/status_change_approved.txt:13]:
```suggestion


```

File: src/registrar/templates/emails/status_change_approved.txt [hunk 4]
```diff
@@ -5,36 +5,39 @@ Congratulations! Your .gov domain request has been approved.
...
+Currently, you’re the only person who can manage this domain. Please keep your contact information updated. 
 
-We strongly recommend that you add other points of contact who will help manage your domain. We also recommend that you provide a security email. This email will allow the public to report security issues on your domain. Security emails are made public. 
-
-Go to the domain management page to add domain contacts <https://manage.get.gov/domain/{{ application.approved_domain.id }}/your-contact-information> and a security email <https://manage.get.gov/domain/{{ application.approved_domain.id }}/security-email>.
+We strongly recommend adding other domain managers who can serve as additional contacts. We also recommend providing a security email that the public can use to report security issues on your domain. You can add domain managers and a security email on the .gov registrar. 
 
-Get help with managing your .gov domain <https://get.gov/help/domain-management/>.
+Learn more about:
+- Adding domain managers <https://beta.get.gov/help/domain-management/#add-a-domain-manager-to-your-.gov-domain>
```
Comment (Katherine-Osos) [inline src/registrar/templates/emails/status_change_approved.txt:30]:
```suggestion
- Adding domain managers <https://get.gov/help/domain-management/#add-a-domain-manager-to-your-.gov-domain>
```

File: src/registrar/templates/emails/status_change_approved.txt [hunk 5]
```diff
@@ -5,36 +5,39 @@ Congratulations! Your .gov domain request has been approved.
...
+- Adding a security email <https://beta.get.gov/help/domain-management/#add-or-update-the-security-email-for-your-.gov-domain>
```
Comment (Katherine-Osos) [inline src/registrar/templates/emails/status_change_approved.txt:31]:
```suggestion
- Adding a security email <https://get.gov/help/domain-management/#add-or-update-the-security-email-for-your-.gov-domain>
```

File: src/registrar/templates/emails/status_change_approved.txt [hunk 6]
```diff
@@ -5,36 +5,39 @@ Congratulations! Your .gov domain request has been approved.
...
+- Domain security best practices <https://beta.get.gov/domains/security/>
```
Comment (Katherine-Osos) [inline src/registrar/templates/emails/status_change_approved.txt:32]:
```suggestion
- Domain security best practices <https://get.gov/domains/security/> 
```

**Comment by Katherine-Osos** [comment]
@michelle-rago I just caught two typos in the "withdrawn" email that you may catch, as well. I updated the file.

- signing into > signing in to
- added a comma within the first sentence of the "Domain Management" section

**Review by michelle-rago — APPROVED** [review]
Looks great, @Katherine-Osos!!

**Pull request merged**